client/dashboard/sites/backups/index.tsx

```diff
@@ -73,16 +89,18 @@ export function BackupsListPage() {
 				/>
 			}
 		>
-			{ hasBackups && (
-				<Grid columns={ 2 }>
-					<BackupsList
-						site={ site }
-						selectedBackup={ selectedBackup }
-						setSelectedBackup={ setSelectedBackup }
-					/>
-					{ selectedBackup && <BackupDetails backup={ selectedBackup } site={ site } /> }
-				</Grid>
-			) }
+			{ hasBackups &&
+				( isSmallViewport ? (
+					<VStack>
+						{ backupDetails }
+						{ backupsList }
+					</VStack>
+				) : (
+					<Grid columns={ 2 }>
+						{ backupsList }
+						{ backupDetails }
+					</Grid>
+				) ) }
```

Comment:
Maybe we could reuse the `Grid` and just change the number of columns depending on the viewport instead of introducing `VStack`. Something like:

```suggestion
			{ hasBackups && (
				<Grid columns={ isSmallViewport ? 1 : 2 }>
					<BackupsList
						site={ site }
						selectedBackup={ selectedBackup }
						setSelectedBackup={ setSelectedBackup }
					/>
					{ selectedBackup && <BackupDetails backup={ selectedBackup } site={ site } /> }
				</Grid>
			) }
```

How does that sound to you?

Reply:
I think that's a cleaner approach and, while it doesn't change the components ordering, it's a reasonable trade-off until we fix the responsive design for good.